src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+
+Responsive design sends the same code and adjusts how the website is displayed based on the screen size, while dynamic serving will send different code depending on the device. The viewport meta tag was used to identify responsive websites vs the Vary: User-Agent header to identify websites using dynamic serving.
+
+ 91.06% of mobile pages include the viewport meta tag, up from 89.16% in 2020. 86.44% of desktop pages also included the viewport meta tag, up from 83.82% in 2020.
+
+{{ figure_markup(
+   image="vary-usage-agent-header-usage.png",
+   caption="`Vary: User-Agent` header usage.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1985736287&format=interactive",
+   sheets_gid="478009067",
+   sql_file="html-response-vary-header-used.sql"
+   )
+}}
+
+For the Vary: User-Agent header, the numbers were pretty much unchanged with 12.6% of desktop pages and 13.4% of mobile pages with this footprint.
+
+One of the biggest reasons for failing mobile-friendliness was that 13.5% of pages did not use a legible font size. Meaning 60% or more of the text had a font size smaller than 12px which can be hard to read on mobile.
+
+### Core Web Vitals
+
+Core Web Vitals are the new speed metrics that are part of Google's Page Experience signals. The metrics measure visual load with Largest Contentful Paint (LCP), visual stability with Cumulative Layout Shift (CLS), and interactivity with First Input Delay (FID).
+
+The data comes from the Chrome User Experience Report (CrUX), which records real-world data from opted-in Chrome users.
+
+33% of mobile websites are now passing Core Web Vitals thresholds, up from 20% last year. Most websites are passing FID but website owners seem to be struggling to improve CLS and LCP.
+
+{{ figure_markup(
+   image="core-web-vitals-2021.png",
+   caption="Core web vitals metrics for 2021.",
+   description="Line chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1693723951&format=interactive",
+   sheets_gid="460991760",
+   sql_file="sql.sql"
+   )
+}}
+
+{# TODO SQL file location #}
+
+See the [Performance](./performance) chapter for more on this topic.
+
+## On-Page
+
+Search engines look at your page's content to determine whether it's a relevant result for the search query. Other on-page elements may also impact rankings or appearance on the search engines.
+
+### Metadata
+
+Metadata includes `<title>` elements and `<meta name="description">` tags. Metadata can directly and/or indirectly affect SEO performance. See the next sections for specifics about each tag.
+
+In 2021, 98.77% of desktop and mobile pages had `<title>` elements. 71.1% of desktop and mobile pages had `<meta name="description">` tags.
```

Comment:
Should not all references to `pages` e.g. `In 2021, 98.77% of desktop and mobile pages had <title> elements` be changed to mobile/desktop ** home** pages to not confuse the reader?

Reply:
```suggestion
In 2021, 98.77% of desktop and mobile pages had `<title>` elements. 71.1% of desktop and mobile homepages had `<meta name="description">` tags.
```